src/main/java/hunternif/mc/atlas/CommonProxy.java

```diff
@@ -30,6 +31,7 @@ public MinecraftServer getServer() {
 	}
 
 	public void preInit(FMLPreInitializationEvent event) {
+		Log.setModLog(event.getModLog());

```

Comment:
As I said in the last PR (https://github.com/AntiqueAtlasTeam/AntiqueAtlas/pull/205) that did something like this, this should use `LogManager.getLogger(...)`. 

Honestly, `Log#modLog` should just be changed to 

```java
private static final Logger LOGGER = LogManager.getLogger(AntiqueAtlasMod.ID);
```

Reply:
That's not exactly, what TehNut meant. Just add a static to the Log class.
`private static final Logger LOGGER = LogManager.getLogger(AntiqueAtlasMod.ID);`

Reply:
Yep, I see that now. Done.